### PR TITLE
Remove router-perf tests

### PR DESCRIPTION
**Pull request by svetsa-rh**
See https://docs.google.com/spreadsheets/d/1TM73n4Y6zKRQjvCX8zFM0LwYxChdsncTEGPHpdsS9Ig/edit?disco=AAAA6KTwnbI

### CODE REVIEW

**Comment by svetsa-rh** [comment]
I'll remove other references of router-perf too. Missed during initial pass here.

**Comment by svetsa-rh** [comment]
@paigerube14 Can you ptal?

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.13-nightly-x86.yaml [hunk 1]
```diff
@@ -88,6 +88,7 @@ tests:
     - chain: workers-scale
     workflow: openshift-qe-node-density-heavy
 - as: perfscale-gcp-network-policy-5nodes-periodic
+  cron: 29 5 */1 * 1-5
```
Comment (paigerube14) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.13-nightly-x86.yaml:91]: do you need to rebase with master, this was part of another pr corrrect

Comment (svetsa-rh) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.13-nightly-x86.yaml:91]: I did rebase. May be it wasn't merged. Will try again.

Comment (svetsa-rh) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.13-nightly-x86.yaml:91]: Odd, it looks like it placed it back

Comment (paigerube14) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.13-nightly-x86.yaml:91]: something seems off here can you try to pull again? 
Can you also try to include this change I mentioned here: https://github.com/openshift/release/pull/43883/files#r1343021128

**Comment by svetsa-rh** [comment]
@paigerube14 I updated network-policy to add always_true: false
Can you ptal?

**Pull request merged**